src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -107,7 +107,7 @@ class PanelMediaBrowser extends LitElement {
         media_content_id: item.media_content_id,
       });
 
-      showMediaPlayerDialog(this, {
+      showMediaPlayerBrowserDialog(this, {
```

Comment:
This should be media player? Not browser?

Reply:
Its the media player in the browser :)

Reply:
maybe it would be better flipped

`showBrowserMediaPlayerDialog`

Reply:
It is a generic media player dialog right? Not browser specific?

Reply:
Its the dialog that plays the media in the browser, if its not the browser as the player then this doesn't get shown

Reply:
WebBrowserPlayMedia